### ISSUE TITLE
Extension breaks web site

### ISSUE THREAD

**Issue by ludacirs** [testing-library/testing-playground#367]
## Bug Report 🐛

The [lawtalk website](https://www.lawtalk.co.kr) breaks when the `testing playground` extension is turned on.

- Testing Playground on 
![image](https://github.com/testing-library/testing-playground/assets/45571631/67b59485-4155-4c49-af91-3081cc77f716)

- Testing Playground off
![image](https://github.com/testing-library/testing-playground/assets/45571631/5b446c1f-29a1-482a-9334-999ca955ebbb)

## To Reproduce ✔️

1. enable Testing Playground extension
2. enter https://www.lawtalk.co.kr
3. you can see - `Uncaught TypeError: Expected an element or document but got string`

## Expected behavior 🤔

Website doesn't break.

## Suggested solution 🔦

I'm not sure why it's breaking, but we've had similar [issues](https://github.com/testing-library/testing-playground/issues/342) in the past.

## Your Environment 💻

- _browser_: Chrome
- _os_: Mac

**Comment by smeijer** [comment]
Yeah, sorry about that. I'll get to it as soon as I have the extension working with manifest 3. Without that, we can't publish an update to the chrome store.
